providers/aws/resources/discovery_conversion.go

```diff
@@ -247,14 +281,17 @@ func addConnectionInfoToEc2Asset(instance *mqlAwsEc2Instance, accountId string,
 	asset.Labels = mapStringInterfaceToStringString(instance.Tags.Data)
 	asset.Name = getInstanceName(instance.InstanceId.Data, asset.Labels)
 	asset.Options = conn.ConnectionOptions()
-	asset.Labels["mondoo.com/region"] = instance.Region.Data
-	asset.Labels["mondoo.com/platform"] = instance.PlatformDetails.Data
-	asset.Labels["mondoo.com/instance-type"] = instance.InstanceType.Data
-	asset.Labels["mondoo.com/parent-id"] = accountId
-	asset.Labels["mondoo.com/instance-id"] = instance.InstanceId.Data
+	addMondooLabels(instanceInfo{
+		region:          instance.Region.Data,
+		platformDetails: instance.PlatformDetails.Data,
+		instanceType:    instance.InstanceType.Data,
+		accountId:       accountId,
+		instanceId:      instance.InstanceId.Data,
+		launchTime:      instance.LaunchTime.Data,
+	}, asset)
 
 	if instance.GetImage().Data != nil {
-		asset.Labels["mondoo.com/image"] = instance.GetImage().Data.Id.Data
+		asset.Labels[MondooImageLabelKey] = instance.GetImage().Data.Id.Data
```

Comment:
Could we also add this to `addMondooLabels` to ave all in one place?

`LaunchTime` also has a `nil` check inside `addMondooLabels`. I guess this would be the same?